projects/controllers/dependencies.py

```diff
@@ -47,13 +47,16 @@ def create_dependency(operator_id, dependency):
     Returns:
         Dependency info.
     """
-
-    dependency = Dependency(uuid=uuid_alpha(),
-                            operator_id=operator_id,
-                            dependency=dependency)
+    if not isinstance(dependency, str):
+        dependency = Dependency(uuid=uuid_alpha(),
+                                operator_id=dependency.operator_id,
+                                dependency=dependency.dependency)
+    else:
+        dependency = Dependency(uuid=uuid_alpha(),
+                                operator_id=operator_id,
+                                dependency=dependency)
```

Comment:
@AlendeGeraldo veja que na documentação deste método está `dependency (str)` mas o código trata `not isinstance(dependency, str)`...não faz sentido. 
```
  Args:
        operator_id (str): the operator uuid.
        dependency **(str)**: the dependency operator uuid.
```

Além disso o código está muito repetitivo:
```
dependency = Dependency(uuid=uuid_alpha(),
                                operator_id=dependency.operator_id,
                                dependency=dependency.dependency)

dependency = Dependency(uuid=uuid_alpha(),
                                operator_id=operator_id,
                                dependency=dependency)

```
basicamente iguais...

## Que tal continuar aceitando só `str` neste método e alterar os métodos que acionam o `create_dependency` para que passem sempre `str`.